app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Recipe Name: <%= @recipe.name %></h1>
+<ul>
+  <li>Complexity: <%= @recipe.complexity %></li>
+  <li>Genre: <%= @recipe.genre %></li>
+  <li>Ingredients:</li>
+    <ul>
+    <% @recipe.ingredients.each do |ingredient| %>
+      <li> <%= ingredient.name %></li>
+    <% end %>
+    </ul>
+</ul>
+<h2>Total Ingredients Cost: $ <%= @ingredients_cost %></h2>
```

Comment:
This is fine, but you can also call `.total_cost` directly here instead of making another instance variable, which would help keep your controllers "skinny".